CHANGES.md

```diff
@@ -1,6 +1,7 @@
 # ChangeLog
 
 ## vX.X.X
+* STIR AcquisitionModel forward and backward signature has changed: default value of `num_subset` and `subset_num` is `None` and the instance properties `num_subset` and `subset_num` will be used in such case. By default these are 0 and 1 respectively, so if the user doesn't change them the behaviour is unchanged.
```

Comment:
```suggestion
* STIR AcquisitionModel `forward`, `direct`, `backward` and `adjoint` signatures have changed in Python. Subset information should now be set via `num_subsets` and `subset_num` members. `The `forward` and `backward` members can still be called with the previous syntax but this will be removed in a later version.
Note that default values of `num_subsets` and `subset_num` are 0 and 1 respectively, such that default behaviour is default behaviour (i.e. process all data) is unchanged.
```